pkg/proxy/nftables/proxier_test.go

```diff
@@ -962,16 +973,14 @@ func TestNodePorts(t *testing.T) {
 
 			fp.syncProxyRules()
 
-			var podIP, externalClientIP, nodeIP, altNodeIP string
+			var podIP, externalClientIP, altNodeIP string
 			if tc.family == v1.IPv4Protocol {
 				podIP = "10.0.0.2"
 				externalClientIP = testExternalClient
-				nodeIP = testNodeIP
 				altNodeIP = testNodeIPAlt
 			} else {
 				podIP = "fd00:10::2"
 				externalClientIP = "2600:5200::1"
-				nodeIP = testNodeIPv6
 				altNodeIP = testNodeIPv6Alt
 			}
 			output := net.JoinHostPort(epIP1, "80") + ", " + net.JoinHostPort(epIP2, "80")
```

Comment:
again, can't comment on the line I want, but there's a comment below

```
			// NodePort on altNodeIP should be allowed, unless
			// nodePortAddressess excludes altNodeIP
```

which is now backwards